extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php

```diff
@@ -32,14 +32,14 @@ public function process( Array $events ) {
 			if ( empty( $emailController ) ) {
 				continue;
 			}
+			Wikia::initAsyncRequest( $wikiId );
```

Comment:
Same question about the call to `Wikia::initAsyncRequest`, is it doing anything necessary? I think the logic in these email controllers doesn't rely on globals.